### PR TITLE
Header

### LINKED ISSUE #93: Header
create the components in the image
image

### PR DESCRIPTION
**Pull request by AlaaShurrab**
fixes #93 #58 
creating the serves display and its components

### CODE REVIEW

File: client/src/pages/LandingPage/index.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
+
 import NavBar from '../../components/NavBar';
 import BookingForm from '../../components/BookingForm';
+import ServesDisplay from '../../components/ServesDisplay';
 import Map from '../home/Map';
 
 const LandingPage = () => (
   <div>
     <NavBar />
+    <ServesDisplay />
```
Comment (Mu7ammadAbed) [inline client/src/pages/LandingPage/index.js:11]:
```suggestion
    <Header />
```

File: client/src/components/ServesCard/style.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.servesCard{
+    width: 240;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    border: 0;
+  }
+
```
Comment (Mu7ammadAbed) [inline client/src/components/ServesCard/style.css:8]: Fix these

File: client/src/pages/LandingPage/index.js (hunk 1 above)
Comment (AlaaShurrab) [reply client/src/pages/LandingPage/index.js:11]: done

File: client/src/components/header/index.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    }
+    servesTitle="Periodontics"
+    servesDescription="Lorem, ipsum dolor sit amet consectetur adipisicing elit. Itaque necessitatibus quidem nesciunt dolor"
+  />,
+];
+
+const Header = () => (
+  <div className="serves-display">
+    <div className="container">
+      <img src={theme} alt="dental cover pic" className="img" />
```
Comment (Mu7ammadAbed) [inline client/src/components/header/index.js:90]: There's a common component we created called Image you should use

File: client/src/components/header/index.js [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+        <div className="content">
+          <h4>Innovative Technology</h4>
+          <h1>Certified dentist</h1>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Itaque
+            necessitatibus quidem nesciunt dolor
+          </p>
+          <button className="read-more-button" type="button">
+            Read More
+          </button>
```
Comment (Mu7ammadAbed) [inline client/src/components/header/index.js:101]: Use the Antd component instead

**Review by Mu7ammadAbed — CHANGES_REQUESTED** [review]

File: client/src/components/ServesCard/index.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Card } from 'antd';
+
+import './style.css';
+
+const { Meta } = Card;
+
+const ServesCard = ({ coverIcon, servesTitle, servesDescription }) => (
```
Comment (Mu7ammadAbed) [inline client/src/components/ServesCard/index.js:10]:
```suggestion
const ServiceCard = ({ coverIcon, serviceTitle, serviceDescription }) => (
```

File: client/src/components/ServesCard/index.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+  <div>
+    <Card
+      hoverable
+      className="servesCard"
```
Comment (Mu7ammadAbed) [inline client/src/components/ServesCard/index.js:14]:
```suggestion
      className="serviceCard"
```

File: client/src/components/ServesCard/index.js [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+      style={{
+        width: 240,
+        display: 'flex',
+        alignItems: 'center',
+        flexFlow: 'column',
+        border: 0,
+      }}
+      cover={coverIcon}
+    >
+      <Meta title={servesTitle} description={servesDescription} />
```
Comment (Mu7ammadAbed) [inline client/src/components/ServesCard/index.js:24]:
```suggestion
      <Meta title={serviceTitle} description={serviceDescription} />
```

File: client/src/components/ServesCard/index.js [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+  </div>
+);
+
+ServesCard.propTypes = {
+  coverIcon: PropTypes.element.isRequired,
+  servesTitle: PropTypes.string.isRequired,
+  servesDescription: PropTypes.string.isRequired,
+};
+
+export default ServesCard;
```
Comment (Mu7ammadAbed) [inline client/src/components/ServesCard/index.js:36]:
```suggestion
ServiceCard.propTypes = {
  coverIcon: PropTypes.element.isRequired,
  serviceTitle: PropTypes.string.isRequired,
  serviceDescription: PropTypes.string.isRequired,
};

export default ServiceCard;
```

File: client/src/components/ServesCard/style.css [hunk 9]
```diff
@@ -0,0 +1,7 @@
+.servesCard{
```
Comment (Mu7ammadAbed) [inline client/src/components/ServesCard/style.css:1]:
```suggestion
.serviceCard{
```

File: client/src/components/header/index.js [hunk 10]
```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+
+import ServesCard from '../ServesCard';
```
Comment (Mu7ammadAbed) [inline client/src/components/header/index.js:3]:
```suggestion
import ServiceCard from '../ServesCard';
```

File: client/src/components/header/index.js [hunk 11]
```diff
@@ -0,0 +1,108 @@
...
+import theme from '../../assets/images/theme1.png';
+import './style.css';
+
+const serves = [
+  <ServesCard
+    key="1"
+    coverIcon={
+      <svg
```
Comment (Mu7ammadAbed) [inline client/src/components/header/index.js:11]: Isn't there any other way to import these svgs from another separate file?

File: client/src/components/header/index.js [hunk 12]
```diff
@@ -0,0 +1,108 @@
...
+        />
+      </svg>
+    }
+    servesTitle="Periodontics"
+    servesDescription="Lorem, ipsum dolor sit amet consectetur adipisicing elit. Itaque necessitatibus quidem nesciunt dolor"
+  />,
+];
+
+const Header = () => (
+  <div className="serves-display">
```
Comment (Mu7ammadAbed) [inline client/src/components/header/index.js:88]:
```suggestion
  <div className="service-display">
```

File: client/src/components/header/index.js [hunk 13]
```diff
@@ -0,0 +1,108 @@
...
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Itaque
+            necessitatibus quidem nesciunt dolor
+          </p>
+          <button className="read-more-button" type="button">
+            Read More
+          </button>
+        </div>
+      </div>
+      <div className="serves-container">{serves}</div>
```
Comment (Mu7ammadAbed) [inline client/src/components/header/index.js:104]:
```suggestion
      <div className="services-container">{services}</div>
```

File: client/src/components/header/index.js [hunk 14]
```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+
+import ServesCard from '../ServesCard';
+import theme from '../../assets/images/theme1.png';
+import './style.css';
+
+const serves = [
+  <ServesCard
```
Comment (Mu7ammadAbed) [inline client/src/components/header/index.js:8]:
```suggestion
const services = [
  <ServiceCard
```

File: client/src/components/header/style.css [hunk 15]
```diff
@@ -0,0 +1,74 @@
+.container{
+    position: relative;
+}
+
+.serves-container{
```
Comment (Mu7ammadAbed) [inline client/src/components/header/style.css:5]:
```suggestion
.services-container{
```

File: client/src/components/header/style.css [hunk 16]
```diff
@@ -0,0 +1,74 @@
...
+    transition: 0.4s;
+
+}
+
+.read-more-button:hover{
+    background-color: black;
+    transition: 0.4s;
+}
+
+.serves-display{
```
Comment (Mu7ammadAbed) [inline client/src/components/header/style.css:72]:
```suggestion
.service-display{
```

**Review by Mu7ammadAbed — CHANGES_REQUESTED** [review]

File: client/src/components/header/index.js (hunk 3 above)
Comment (AlaaShurrab) [reply client/src/components/header/index.js:90]: done

File: client/src/components/header/index.js (hunk 4 above)
Comment (AlaaShurrab) [reply client/src/components/header/index.js:101]: done

File: client/src/components/ServesCard/index.js (hunk 5 above)
Comment (AlaaShurrab) [reply client/src/components/ServesCard/index.js:10]: done

File: client/src/components/ServesCard/index.js (hunk 6 above)
Comment (AlaaShurrab) [reply client/src/components/ServesCard/index.js:14]: done

File: client/src/components/ServesCard/index.js (hunk 7 above)
Comment (AlaaShurrab) [reply client/src/components/ServesCard/index.js:24]: done

File: client/src/components/ServesCard/index.js (hunk 8 above)
Comment (AlaaShurrab) [reply client/src/components/ServesCard/index.js:36]: done

File: client/src/components/ServesCard/style.css (hunk 9 above)
Comment (AlaaShurrab) [reply client/src/components/ServesCard/style.css:1]: done

File: client/src/components/header/index.js (hunk 10 above)
Comment (AlaaShurrab) [reply client/src/components/header/index.js:3]: done

File: client/src/components/header/index.js (hunk 12 above)
Comment (AlaaShurrab) [reply client/src/components/header/index.js:88]: done

File: client/src/components/header/index.js (hunk 13 above)
Comment (AlaaShurrab) [reply client/src/components/header/index.js:104]: done

File: client/src/components/header/index.js (hunk 14 above)
Comment (AlaaShurrab) [reply client/src/components/header/index.js:8]: done

File: client/src/components/header/style.css (hunk 15 above)
Comment (AlaaShurrab) [reply client/src/components/header/style.css:5]: done

File: client/src/components/header/style.css (hunk 16 above)
Comment (AlaaShurrab) [reply client/src/components/header/style.css:72]: done

**Pull request closed without merging**